src/Views.elm

```diff
@@ -34,21 +36,33 @@ appBody : Model -> Html Msg
 appBody model =
     case model.route of
         StatusRoute ->
-            Status.Views.view model
+            Views.Status.Views.view model
+
+        SilenceRoute silenceId ->
+            Views.Silence.Views.view model
 
         AlertsRoute route ->
             case model.alertGroups of
                 Success alertGroups ->
-                    Html.map alertTranslator (Alerts.Views.view route alertGroups model.filter (text ""))
+                    Views.AlertList.Views.view route alertGroups model.filter (text "")
 
                 Loading ->
                     loading
 
                 Failure msg ->
-                    Html.map alertTranslator (Alerts.Views.view route [] model.filter (error msg))
+                    Views.AlertList.Views.view route [] model.filter (error msg)
+
+        SilenceListRoute route ->
+            Views.SilenceList.Views.view model.silences model.silence model.currentTime model.filter
+
+        SilenceFormNewRoute ->
+            Views.SilenceForm.Views.newForm model
```

Comment:
In the long run I would prefer to have separate views for the edit and new which both share the basic input form code but for example have different buttons (e.g. "Create" vs. "Update" buttons).